figures/Fig3ideas.Rmd

```diff
@@ -0,0 +1,97 @@
+---
+title: "Figure 3 ideas"
+author: "Sara Gosline"
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    code_folding: hide
+    toc: true
+    toc_float: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+library(synapser)
+synLogin()
+require(tidyverse)
+
+
+```
+
+## Grab data from table
+The goal of this figure is to summarize the results in `syn21326918`.
+
+```{r rna-Seq, warning=FALSE}
+
+rf.results<-synTableQuery('select * from syn21326918')$asDataFrame()%>%
+  select(-c(ROW_ID,ROW_VERSION))
+
+DT::datatable(rf.results)
+
+```
+
+## Figure 3A
+
+Now we can go ahead and plot a venn diagram of the RF results. The first thing we want to do is evaluate how many latent variables predict which tumor types. This Venn Diagram illustrates that there are numerous LVs that predict many different tumor types. 
```

Comment:
this paper is a great Vennue for this sort of plot

Reply:
lolol